include/aws/common/cpu_usage_sampler.h

```diff
@@ -0,0 +1,52 @@
+#ifndef AWS_COMMON_CPU_USAGE_SAMPLER_H
+#define AWS_COMMON_CPU_USAGE_SAMPLER_H
```

Comment:
trivial/debatable: this doesn't need to be its own header, the aws_cpu_sampler stuff would fit in fine with the other CPU functions in system_info.h